api/src/pcapi/core/educational/api/offer.py

```diff
@@ -1149,48 +1149,71 @@ def batch_update_collective_offers_template(query: BaseQuery, update_fields: dic
+                educational_models.CollectiveStock.startDatetime < datetime.datetime.utcnow(),
+            )
+            .count()
         )
-        .count()
-    )
-    if count_started_stocks > 0:
-        raise offers_exceptions.OfferEventInThePast(count_started_stocks)
-
-    count_reimbursed_bookings = (
-        educational_models.CollectiveBooking.query.with_entities(educational_models.CollectiveBooking.id)
-        .join(educational_models.CollectiveBooking.collectiveStock)
-        .filter(
-            educational_models.CollectiveStock.collectiveOfferId == collective_offer.id,
-            educational_models.CollectiveBooking.isReimbursed,
+        if count_started_stocks > 0:
+            raise offers_exceptions.OfferEventInThePast(count_started_stocks)
+
+        count_reimbursed_bookings = (
+            educational_models.CollectiveBooking.query.with_entities(educational_models.CollectiveBooking.id)
+            .join(educational_models.CollectiveBooking.collectiveStock)
+            .filter(
+                educational_models.CollectiveStock.collectiveOfferId == collective_offer.id,
+                educational_models.CollectiveBooking.isReimbursed,
+            )
+            .count()
         )
-        .count()
+        if count_reimbursed_bookings > 0:
+            raise offers_exceptions.OfferHasReimbursedBookings(count_reimbursed_bookings)
+
+    venues_choices = offerers_repository.get_offerers_venues_with_pricing_point(
+        collective_offer.venue,
+        include_without_pricing_points=not with_restrictions,
+        only_similar_pricing_points=not with_restrictions,
+        filter_same_bank_account=not with_restrictions,
     )
-    if count_reimbursed_bookings > 0:
-        raise offers_exceptions.OfferHasReimbursedBookings(count_reimbursed_bookings)
-
-    venues_choices = offerers_repository.get_offerers_venues_with_pricing_point(collective_offer.venue)
     if not venues_choices:
         raise offers_exceptions.NoDestinationVenue()
     return venues_choices
 
 
 def move_collective_offer_venue(
-    collective_offer: educational_models.CollectiveOffer, destination_venue: offerers_models.Venue
+    collective_offer: educational_models.CollectiveOffer,
+    destination_venue: offerers_models.Venue,
+    with_restrictions: bool = True,
 ) -> None:
-    venue_choices = check_can_move_collective_offer_venue(collective_offer)
+    venue_choices = check_can_move_collective_offer_venue(collective_offer, with_restrictions=with_restrictions)
 
     if destination_venue not in venue_choices:
         raise offers_exceptions.ForbiddenDestinationVenue()
 
     destination_pricing_point_link = destination_venue.current_pricing_point_link
-    assert destination_pricing_point_link  # for mypy - it would not be in venue_choices without link
-    destination_pricing_point_id = destination_pricing_point_link.pricingPointId
+    if not with_restrictions:
```

Comment:
[very low logaf] pour les prochaines fois quand c'est possible, c'est toujours plus rapide de comprendre un 
```suggestion
    if with_restrictions:
```
que l'inverse 😉

Reply:
(surtout dans `get_offerers_venues_with_pricing_point` , c'est moins évident à relire avec les `not with_restrictions`)

Reply:
En effet, l'objectif ici était de ne modifier le moins possible le code existant et de "juste" ajouter les modifications pour les besoin aux endroits nécessaire.